Content.Server/Body/Systems/RespiratorSystem.cs

```diff
@@ -104,6 +105,9 @@ public override void Update(float frameTime)
 
             if (respirator.Saturation < respirator.SuffocationThreshold)
             {
+                if (TryComp<CosmicCultComponent>(uid, out var cultComponent) && !cultComponent.Respiration && !_mobState.IsIncapacitated(uid)) return; // DeltaV: Cosmic Cult
+                // One line change but a refactor would be better. this is kinda cringe. Makes cultists gasp and respirate but not asphyxiate in space.
```

Comment:
Comment should be before the change
```suggestion
                // DeltaV - One line change but a refactor would be better. this is kinda cringe. Makes cosmic cultists gasp and respirate but not asphyxiate in space.
                if (TryComp<CosmicCultComponent>(uid, out var cultComponent) && !cultComponent.Respiration && !_mobState.IsIncapacitated(uid)) return;
```